features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin_new/source/src/app/components/Throttling/Blacklist/List.jsx

```diff
@@ -95,6 +98,77 @@ export default function ListBlacklistThrottlingPolicies() {
         }),
     };
 
+    /**
+ * Mock API call
+ * @returns {Promise}.
+ */
+    function apiCall() {
+        let policyList;
+        let incrementId = 0;
```

Comment:
Same as above , no need to declare the variable outside the lambda function scope right ? `incrementId `

Reply:
fixed